developers/wcs/platform/billing.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Billing
+sidebar_position: 3
+# image: og/wcs/user_guides.jpg
+---
+
+Sandbox clusters are free. You do not need a billing account to create a Sandbox cluster.
+
+Serverless clusters are charged for resource usage. The monthly charge is determined by your usage and your support level. You have to have a billing account to create a serverless cluster.
+
+To estimate costs for different combinations of usage levels and SLAs, see the [pricing calculator](https://weaviate.io/services/serverless).
+
+## Create an account
+
+Weaviate Cloud (WCD) uses [Stripe](https://stripe.com/) for billing. To create a Stripe account follow these steps:
+
+1. On the Clusters tab, click the `Create cluster` button.
+1. Choose a Managed cluster.
+1. Click the `Add billing` button.<br/>
+1. Select your location on the next screen.
+1. Continue to the Stripe account creation page.<br/>
+1. Create your Stripe account.
+
+## Usage charges
+
+Usage charges reflect the number and size of the objects in your collection. Weaviate polls each collection to determine a daily object count.
+
+- If the objects are uncompressed, the billing system multiplies the object count by the number of dimensions to get the `vectorDimensionsSum`.
+- If the objects are compressed, the billing system multiplies the object count by the number of vector segments to get the `vectorSegmentsSum`.
+- Older plans may also calculate a similar value, the `queryDimensionsTotal`.
+
+The values are added together and normalized to get a base price. The base price is multiplied by the pricing rate for the cluster's SLA tier.
+
+If the cluster has high availability, the pricing rate is multiplied by three.
+
+There are significant cost savings when compression is enabled.
+
+## Support levels
+
+Weaviate offers three levels of support, [Standard](/developers/wcs/platform/support-levels#standard-support), [Professional](/developers/wcs/platform/support-levels#professional-support), and [Business Critical](/developers/wcs/platform/support-levels#business-critical-support). Each support level (SLA) has a billing rate.
+
+The billing rate is a minimum charge per month.
```

Comment:
It is both a minimum charge per month as well as a pricing rate per plan

Reply:
WIth the plans, standard is the only plan a user can create for themselves through the UI. Other plans they need to reach out to us and have them created internally. I believe the rates and minimums are public, as we have the published pricing calculator, might be good to link that here https://weaviate.io/pricing

Reply:
This is the tricky part; we are telling a mixed story. Our billing in really usage*SLA but we've been telling people it's usage based. In effect we've only been pushing part of the story.

Updated